README.md

```diff
@@ -21,3 +21,44 @@ The project requires some extensions of the default VCL provided by Helix. The p
 ```
 
 The patched version of the 3 subroutines adds the parsing of the host url to extract the content owner / repo that would override the one stored in the Fastly dictionary.
+
+## Automatic publishing
+
+Go to your Google Drive account
+ * https://drive.google.com/drive/my-drive
+
+Create a new shared folder to hold the website root
+ * Click on the button “+ New” > Folder
+ * Give the folder a name (e.g., sitename)
+ * Double-click the sitename folder to open
+
+Share the folder with helix
+ * Select the menu arrow after MyDrive > sitename > Share with others ...
+ * Share it with helix.integration@gmail.com
+ * Give helix write permission (necessary for ???)
+ * Copy the folder URL (e.g., https://drive.google.com/drive/folders/{id} )
+
+Create a domain mount point on GitHub
+ * Go to your GitHub home (e.g., https://github.com/username/ )
+ * Select Repositories tab and green New button
+ * Create a new repository
+   * Give it a short name (for the domain)
+   * Make it Public
+ * Inside repository, create a new file “fstab.json” with content:
+```
+{ "mountpoints": [ {
+      "root": "/",
```

Comment:
Using `/` can be expensive because it would always try to fetch GH assets like `header.md` from the mount first, then fall back to GH. Until that's improved, I'd use something like `/content`:
```suggestion
      "root": "/content",
```